crud/select/compat/common.lua

```diff
@@ -1,6 +1,32 @@
+local ratelimit = require('crud.ratelimit')
+local check_select_safety_rl = ratelimit.new()
+
 local common = {}
 
 common.SELECT_FUNC_NAME = '_crud.select_on_storage'
 common.DEFAULT_BATCH_SIZE = 100
 
+common.check_select_safety = function(space_name, user_conditions, opts)
+    if opts.fullscan ~= nil and opts.fullscan == true then
+        return
+    end
+
+    if opts.first ~= nil and math.abs(opts.first) <= 1000 then
+        return
+    end
+
+    if user_conditions ~= nil then
+        for _, v in ipairs(user_conditions) do
+            local it = v[1]
+            if it ~= nil and type(it) == 'string' and (it == '=' or it == '==') then
+                return
+            end
+        end
+    end
+
+    local rl = check_select_safety_rl
+    local traceback = debug.traceback()
+    rl:log_crit("Potentially long select from space '%s'\n %s", space_name, traceback)
```

Comment:
This message is a bit non-informative: it is hard to understand what should we do to remove potentially long selects from the application. Even though I don't have any alternatives for the message, I think we should document warning conditions in README so developer may understand what really "potentially long select" is.

Reply:
The message is copy-paste from the Tarantool implementation:
https://github.com/tarantool/tarantool/pull/7131#pullrequestreview-972344813

I will add a note to README.

Reply:
Oh, ok. Sorry, I have only seen the previous version 
![image](https://user-images.githubusercontent.com/20455996/169512541-779f5083-6831-42b6-81bc-b21faf2fd8bf.png)